backend/services/engine.py

```
@@ -88,6 +90,49 @@ def playback_worker(
+            return audio_bytes
+
+        if not audio_bytes:
+            return audio_bytes
+
+        samples = np.frombuffer(audio_bytes, dtype=np.int16)
+        if samples.size == 0:
+            return audio_bytes
+
+        scaled = np.clip(samples.astype(np.float32) * level, -32768, 32767).astype(np.int16)
+        return scaled.tobytes()
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)